staging/src/k8s.io/apiserver/pkg/admission/interfaces.go

```diff
@@ -49,6 +49,13 @@ type Attributes interface {
 	GetKind() schema.GroupVersionKind
 	// GetUserInfo is information about the requesting user
 	GetUserInfo() user.Info
+	// SetAnnotations sets annotation key-value pairs in a structed format:
+	// "<plugin_name>/<key>"="value"
+	// plugin_name should be fully qualified, e.g., podsecuritypolicy.admission.k8s.io, where
+	// "podsecuritypolicy" is the name of the plugin, and admission.k8s.io is the name
+	// of the organization.
+	// Both ValidationInterface and MutationInterface are allowed to set Annotations.
+	SetAnnotations(plugin_name, key, value string) bool
```

Comment:
pluginName or even only plugin. Underscore is against convention.

Reply:
Done.